client/src/styles/Landing/LandingFormStyles.js

```diff
@@ -0,0 +1,18 @@
+const LandingFormStyles = () => ({
+  root: {
+    backgroundColor: '#FFFFFF',
+    width: 600,
+    borderRadius: 7,
+    marginTop: 60,
+    padding: '64px 0 64px 0'
```

Comment:
Thanks for bringing this to my attention. The issue was caused by the default `wrap` property of Material-UI's `<Grid>` component. If the viewport was too short to display the whole form, it would try to wrap around and start a new column. I've pushed a fix for this, please let me know if it resolves the issue.

Reply:
That fixed it. Thanks Danny!